<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonEnable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonDisable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Выключить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonpause">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Пауза</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEmergency">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Экстренное 
 торможение </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Состояние:</string>
    </property>
   </widget>
   <widget class="QLabel" name="outState">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выключен</string>
    </property>
   </widget>
   <widget class="QPushButton" name="handauto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>81</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>Смена 
 режима </string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>591</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ручной режим</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_plus_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_plus_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_plus_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_plus_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_plus_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_plus_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_minus_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_minus_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_minus_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_minus_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_minus_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Axis_minus_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>_</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ось 1 </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ось 2 </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ось 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ось 4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ось 5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ось 6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changemode">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CARTESIAN MODE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Текущий режим</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushobject">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Схват</string>
      </property>
     </widget>
     <widget class="QPushButton" name="movetostart">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Возвращение на старт</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Автоматический режим </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
